### PR TITLE
Ansible documentation 0.x

### PR DESCRIPTION
**Pull request by dLobatog**
Aside from the manual, it also changes some parts of the front page and
the manual in order to acknowledge Ansible reporting and facts.

### CODE REVIEW

**Comment by dLobatog** [comment]
Pending https://github.com/theforeman/foreman-packaging/pull/1012

File: plugins/foreman_ansible/0.2/index.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+In order to make Ansible send us data from the hosts, we set up a callback on your host that runs Ansible. To do this:
+
+* Change your /etc/ansible/ansible.cfg to contain the following options
+
+      callback_plugins = ~/.ansible/plugins/callback_plugins/
+      bin_ansible_callbacks = True
+
+* And run this line to download the callback file in the appropriate location
+
+      wget https://raw.githubusercontent.com/theforeman/foreman_ansible/master/extras/foreman_callback.py -O ~/.ansible/plugins/callback_plugins/foreman_callback.py
```
Comment (mmoll) [inline plugins/foreman_ansible/0.2/index.md]: maybe use the version here?

File: plugins/foreman_ansible/0.2/index.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+* Now open the script to set up `FOREMAN_URL` and `FOREMAN_SSL_CERT` variables on the callback. Alternatively, you can find the source for the [Ansible callback here](https://github.com/theforeman/foreman_ansible) and deploy it in your directory of choice.
+
+## 2.2 Plugin
+### 2.2.1 Installer (recommended on Foreman 1.11+)
```
Comment (mmoll) [inline plugins/foreman_ansible/0.2/index.md]: I'd rather wait for this, unitl it's clear (and merged into -installer) what's needed on the core and and the proxy side. At least it should be clear that there is a distinction between core and proxy.

File: plugins/foreman_ansible/0.2/index.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+* Change your /etc/ansible/ansible.cfg to contain the following options
+
+      callback_plugins = ~/.ansible/plugins/callback_plugins/
+      bin_ansible_callbacks = True
+
+* And run this line to download the callback file in the appropriate location
+
+      wget https://raw.githubusercontent.com/theforeman/foreman_ansible/master/extras/foreman_callback.py -O ~/.ansible/plugins/callback_plugins/foreman_callback.py
+
+* Now open the script to set up `FOREMAN_URL` and `FOREMAN_SSL_CERT` variables on the callback. Alternatively, you can find the source for the [Ansible callback here](https://github.com/theforeman/foreman_ansible) and deploy it in your directory of choice.
```
Comment (mmoll) [inline plugins/foreman_ansible/0.2/index.md]: a link or more information regarding SSL certs would be nice. I suppose a trusted cert is needed?

File: plugins/foreman_ansible/0.2/index.md (hunk 2 above)
Comment (dLobatog) [inline plugins/foreman_ansible/0.2/index.md]: On the proxy side - nothing - there's no proxy component for the moment, at least as far as foreman_ansible is concerned. Remote execution is a whole another thing that is not merged yet. But I'll remove the installer section until 1.11 then!

File: plugins/foreman_ansible/0.2/index.md (hunk 3 above)
Comment (dLobatog) [inline plugins/foreman_ansible/0.2/index.md]: Yup, not sure what else can I write..?

**Comment by dLobatog** [comment]
I updated this except for the SSL certs part - not sure what else to comment there

File: plugins/foreman_remote_execution/0.2/index.md [hunk 4]
```diff
@@ -31,7 +31,7 @@ A few examples of tasks that this plugin helps you achieve:
 
 - Install/update/remove a package
 - Bootstrap a configuration management agent
-- Trigger a Puppet/Salt/Chef run
+- Trigger a Puppet/Salt/Chef/Ansible run
```
Comment (domcleal) [inline plugins/foreman_remote_execution/0.2/index.md]: Note that the foreman_remote_execution plugin maintains a nightly/ version that's used to cut their next release, you should probably update that (either instead, or as well as, if they want this change).

Comment (dLobatog) [inline plugins/foreman_remote_execution/0.2/index.md]: Good call actually, it's not even merged yet

**Comment by dLobatog** [comment]
Added a bit about configuring SSL certs and URL via environment variables thanks to @agx 's latest addition to the plugin

File: plugins/foreman_ansible/0.x/index.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+In order to make Ansible send us data from the hosts, we set up a callback on your host that runs Ansible. To do this:
+
+* Change your /etc/ansible/ansible.cfg to contain the following options
+
+      callback_plugins = ~/.ansible/plugins/callback_plugins/
+      bin_ansible_callbacks = True
+
+* And run this line to download the callback file in the appropriate location
+
+      wget https://raw.githubusercontent.com/theforeman/foreman_ansible/0.2.2/extras/foreman_callback.py -O ~/.ansible/plugins/callback_plugins/foreman_callback.py
```
Comment (mmoll) [inline plugins/foreman_ansible/0.x/index.md]: 0.3?

**Comment by dLobatog** [comment]
@mmoll Updated to point the callback file to 0.3, thanks !

**Comment by dLobatog** [comment]
@mmoll Actually ended up changing it to master instead of 0.3. There are a few key changes in master that fix bugs with Ansible Tower & default SSL setup - https://github.com/theforeman/foreman_ansible/commits/master/extras/foreman_callback.py

**Comment by mmoll** [comment]
merged, gracias @dLobatog!

**Pull request closed without merging**